src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/request/StrikeOffObjectionsRequestEntity.java

```diff
@@ -0,0 +1,129 @@
+package uk.gov.companieshouse.api.strikeoffobjections.model.request;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
+
+import java.time.LocalDateTime;
+
+@Document(collection = "strike_off_objections_requests")
+public class StrikeOffObjectionsRequestEntity {
+
+
+    public static class Builder {
+        private String requestId;
```

Comment:
And this is an 'objectionId' as per @markpit 's API spec PR.

Reply:
I think it's just 'id' in the API spec